bigtop.bom

```diff
@@ -304,11 +304,10 @@ bigtop {
        * to the base HBase version. Update as needed whenever changing the
        * HBase version in the BOM.
        */
```

Comment:
@sekikn You can remove this comment before merging.

Reply:
Just removed, thanks @iwasakims!